components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +433,14 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        try {
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(Double.parseDouble(reCaptchaScoreThreshold));
+        } catch (NumberFormatException | NullPointerException e) {
+            log.info("Error parsing recaptcha.threshold from config", e);
```

Comment:
```suggestion
            log.info("Error parsing recaptcha.threshold from config", e);
```log.info("Error while parsing recaptcha score threshold value. Hence using the default value :  x", e);